drawing_5.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<head>
+<html lang='en'>
+	<meta charset="UTF-8">
+		<title>Drawing 5</title>
+			<link rel="stylesheet" type="text/css" href="drawing_5.css">
+</head>
+
+<body>
+	<div class="one">
+		<div class="single"></div>
```

Comment:
don't need a nested inner `div` here, outer should work fine, as long as you set `width` and `height`